bindings-go/main/main.go

```diff
@@ -0,0 +1,126 @@
+package main
```

Comment:
this should be part of the examples folder not the main method for the go binding